lib/js2i18next.js

```diff
@@ -19,7 +19,7 @@ const emptyOrObject = (key, value, options) => {
 }
 
 const getI18nextPluralExtension = (ext, i) => {
-  if (ext && ext.numbers && ext.numbers.length === 2) return i === 0 ? '' : '_plural'
+  if (ext && ext.numbers && ext.numbers.length <= 2) return i === 0 ? '' : '_plural'
```

Comment:
This is the real change. We still want the `singular` version for keys using plurals if the language does not have plural.